apps/omg_watcher/test/omg_watcher/exit_processor/core_test.exs

```diff
@@ -38,6 +38,35 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
   @utxo_pos2 Utxo.position(@late_blknum - 1_000, 0, 1)
 
   describe "generic sanity checks" do
+    test "can initialise with standard exits that do not have recently introduced keys" do
+      recent_keys = [:root_chain_txhash]
+      utxo_position = {0, 0, 0}
+
+      exit_info = %{
+        amount: 1,
+        currency: random_bytes(20),
+        eth_height: 1,
+        exit_id: 1,
+        exiting_txbytes: random_bytes(32),
+        is_active: false,
+        owner: random_bytes(20)
+      }
+
+      {:ok, %{exits: exits}} =
+        Core.init(
+          [
```

Comment:
pls fix this formatting

Reply:
Believe this is an old commit. No changes to `core_test.exs`.